diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -16,17 +19,27 @@ import org.cqfn.diktat.ruleset.rules.identifiers.LocalVariablesRule
 import org.cqfn.diktat.ruleset.rules.kdoc.KdocComments
 import org.cqfn.diktat.ruleset.rules.kdoc.KdocFormatting
 import org.cqfn.diktat.ruleset.rules.kdoc.KdocMethods
+import org.cqfn.diktat.ruleset.utils.editorDistance
 import org.slf4j.LoggerFactory
+import java.io.File
 
 /**
  * this constant will be used everywhere in the code to mark usage of Diktat ruleset
  */
 const val DIKTAT_RULE_SET_ID = "diktat-ruleset"
 
 class DiktatRuleSetProvider(private val diktatConfigFile: String = "diktat-analysis.yml") : RuleSetProvider {
+    @Suppress("LongMethod")
     override fun get(): RuleSet {
         log.debug("Will run $DIKTAT_RULE_SET_ID with $diktatConfigFile (it can be placed to the run directory or the default file from resources will be used)")
-        val configRules = RulesConfigReader(javaClass.classLoader).readResource(diktatConfigFile) ?: listOf()
+        if (!File(diktatConfigFile).exists()) {
+            log.warn("File $diktatConfigFile not found in file system, the file included in jar will be used. " +
```

Comment:
I guess we need explicitly say "Be careful! File $diktatConfiFile not found ... It should be placed in the directory where diktat binary is placed "

Reply:
Yes, I'll change warning message. Btw, for maven plugin user can specify arbitrary config location, not necessary WD.